accelerator/core/core.c

```diff
@@ -518,12 +518,12 @@ status_t ta_get_bundles_by_addr(const iota_client_service_t* const service, tryt
   }
 
   iota_transaction_t* curr_tx = NULL;
+  bundle_transactions_t* bundle = NULL;
   TX_OBJS_FOREACH(obj_res, curr_tx) {
+    bundle_transactions_new(&bundle);
```

Comment:
Is it ok that call `bundle_transactions_new(&bundle)` without free?

Reply:
we free bundle at line 536

Reply:
Oh, I miss it.